### PR TITLE
Add events sensor to hold events per device

### PR DESCRIPTION
**Pull request by samuolis**
- Added events sensor which will hold events attributes.
- Attributes will be separated by type:
  - user actions
  - alarms
  - other
- Updated readme to explain how to render events in lovelace.

Example:
Screenshot 2022-02-20 at 17 36 50

### CODE REVIEW

**Comment by Zhalias** [comment]
Ačiū, bandysiu aiškintis.
-------- Originali žinutė --------Nuo: Lukas Samuolis ***@***.***> Data: 2022-02-20  18:08  (GMT+02:00) Kam: augustas2/eldes ***@***.***> Kopija: Subscribed ***@***.***> Tema: [augustas2/eldes] Add events sensor to hold events per device (PR #8) 
Added events sensor which will hold events attributes.
Attributes will be separated by type:

user actions
alarms
other


Updated readme to explain how to render events in lovelace.

Example:




You can view, comment on, or merge this pull request online at:
  https://github.com/augustas2/eldes/pull/8

Commit Summary

  f5770c4  Add events sensor
  5c39a8b  Update README.md
  8afafd5  Update README.md
  e87a91e  Update README.md


File Changes  (5 files)

  
    M
    README.md
    (41)
  
  
    M
    custom_components/eldes_alarm/__init__.py
    (1)
  
  
    M
    custom_components/eldes_alarm/const.py
    (4)
  
  
    M
    custom_components/eldes_alarm/core/eldes_cloud.py
    (15)
  
  
    M
    custom_components/eldes_alarm/sensor.py
    (85)
  


Patch Links:

  https://github.com/augustas2/eldes/pull/8.patch
  https://github.com/augustas2/eldes/pull/8.diff

File: README.md [hunk 1]
```diff
@@ -17,6 +17,47 @@ You have two options for installation:
...
+- `alarms` -> holds events with alarms
+- `user_actions` -> holds events with user action like arm and disarm
+- `events` -> holds all other events
+
+#### Display events
+
+To display events used another integration:
+https://github.com/gadgetchnnel/lovelace-home-feed-card
+
+#### Example config for lovelace-home-feed-card
```
Comment (augustas2) [inline README.md]: Add a link to `lovelace-home-feed-card` integration if possible.

File: custom_components/eldes_alarm/sensor.py [hunk 2]
```diff
@@ -11,7 +12,7 @@
     DATA_COORDINATOR,
     DOMAIN,
     SIGNAL_STRENGTH_MAP,
-    BATTERY_STATUS_MAP
+    BATTERY_STATUS_MAP, ATTR_EVENTS, ATTR_ALARMS, ATTR_USER_ACTIONS
```
Comment (augustas2) [inline custom_components/eldes_alarm/sensor.py]: Keep every variable in separate line. Just for same style.

File: custom_components/eldes_alarm/sensor.py [hunk 3]
```diff
@@ -146,3 +148,84 @@ def native_value(self):
...
+        return f"{self.imei}_events"
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return f"Events"
+
+    @property
+    def native_value(self):
+        """Return the state of the sensor."""
```
Comment (augustas2) [inline custom_components/eldes_alarm/sensor.py]: Change comment to - """Return the value of the sensor."""

**Review by augustas2 — CHANGES_REQUESTED** [review]
Small changes needed before merge.

File: README.md (hunk 1 above)
Comment (samuolis) [reply README.md]: Added above, but will try to add as hyperlink

**Review by augustas2 — APPROVED** [review]

**Pull request merged**